homework_4/bet_acceptance_api/internal/infrastructure/rabbitmq/betPublisher.go

```diff
@@ -0,0 +1,70 @@
+package rabbitmq
+
+import (
+	"encoding/json"
+	"log"
+
+	"github.com/jelenanemcic/code-cadets-2021/homework_4/bet_acceptance_api/internal/infrastructure/rabbitmq/models"
+	"github.com/streadway/amqp"
+)
+
+const contentTypeTextPlain = "text/plain"
```

Comment:
Doesn't really matter (since rabbit MQ doesn't use content-type internally at all, I think) but "application/json" might be a bit more accurate here.